tools/standalone_install.sh

```diff
@@ -74,8 +74,7 @@ if [ ! $(which go) ]; then
     exit 1
 fi
 go_version=$(go version)
-# This expression will need to be updated for future releases.
-echo $go_version | grep -q -E 'go version go1\.5'
+echo $go_version | grep -E -q --regexp="go[1-9]\.[5-9]"
```

Comment:
this will break for version `[2-9]\.[0-4]` and doesn't allow go version > 1.9 (eg. 1.10)

Since we're now at version 1.6, and ubuntu supports 1.5, can we drop this test entirely? @ameihm0912 ?